pkg/rancher-desktop/backend/lima.ts

```diff
@@ -753,9 +784,16 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
    * Run the given command within the VM.
    */
   limaSpawn(options: execOptions, args: string[]): ChildProcess {
+    if (options.root) {
+      args = ['sudo'].concat(args);
+    }
     args = ['shell', '--workdir=.', MACHINE_NAME].concat(args);
     args = this.debug ? ['--debug'].concat(args) : args;
```

Comment:
You can move `args.unshift('debug')` inside the `if (this.debug)` test unless you think the log stmt is temporary.

Reply:
Done and amended in f363bc2f175535a251c9912abc3084b467b36439